ui/desktop/app/templates/scopes/scope/projects/settings/index.hbs

```diff
@@ -0,0 +1,63 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+<SettingsCard @header={{t 'titles.application'}} @icon='boundary'>
+  <div class='content'>
+    <div class='section'>
```

Comment:
Can we ellaborate on the names we are assigning to clases here? `content` is clear, but then `section` and after `theme`? seems to me these is quite ad-hoc, but this is a pattern that we will reproduce in other settings cards, can we look at something more generic?

Reply:
agreed it could be better, I updated this